funcsigs/__init__.py

```diff
@@ -20,8 +20,8 @@
 __all__ = ['BoundArguments', 'Parameter', 'Signature', 'signature']
 
 
-_WrapperDescriptor = type(type.__call__)
-_MethodWrapper = type(all.__call__)
+_WrapperDescriptor = type(bytearray.__add__)
+_MethodWrapper = type(u'str'.__add__)
```

Comment:
Why not plain `'str'.__add__`? `u''` introduces an incompatibility with Python 3.2